tldw_Server_API/app/api/v1/schemas/identity_provider_schemas.py

```
@@ -0,0 +1,184 @@
+
+    @field_validator("issuer")
+    @classmethod
+    def normalize_issuer(cls, value: str) -> str:
+        return value.strip()
+
+    @field_validator("owner_scope_id")
+    @classmethod
+    def validate_owner_scope_id(cls, value: int | None, info) -> int | None:
+        owner_scope_type = info.data.get("owner_scope_type", "global")
+        if owner_scope_type == "org" and value is None:
+            raise ValueError("owner_scope_id is required for org-scoped providers")
```

Ruff (0.15.6)

[warning] Avoid specifying long messages outside the exception class

(TRY003)